docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -71,7 +71,7 @@ to generate the `extraData` RLP string to include in the genesis file.
     besu rlp encode --from=toEncode.json
     ```    
 
-Where the `toEncode.json` file contains a list of the initial validators in ascending order. 
+Where the `toEncode.json` file contains a list of the initial validators in ascending order. Use the Besu subcommand [`public-key export-address`](../../../Reference/CLI/CLI-Subcommands.md#export-address) to write the address of a validator to the `toEncode.json` file. Modify the file to conform to JSON input format.
```

Comment:
What if there is more than one validator?  Will this workflow work with multiple nodes?

Reply:
It would not work. I revised the new content so that users won't write the address to the `toEncode.json` file.